### ISSUE TITLE
Issue with lua-devel on RHEL 9.1

### ISSUE THREAD

**Issue by nortex** [idiv-biodiversity/ansible-role-lmod#16]
Hello, i am trying to use the role, but it fails on package lua-devel. 

It seems that role adds the default epel repo where no lua-devel package persists. Altrough it is tagged to use crb repo , i can see it is not really adding the crb repo.

**Comment by wookietreiber** [comment]
@nortex We currently don't have a RHEL 9 yet to test on, we only tested on Rocky 9. Unfortunately, I can't help with that right now, until I manage to get RHEL dev subscription to test the role on RHEL.

**Comment by nortex** [comment]
> @nortex We currently don't have a RHEL 9 yet to test on, we only tested on Rocky 9. Unfortunately, I can't help with that right now, until I manage to get RHEL dev subscription to test the role on RHEL.

Thanks for the response.

I have found where lua-devel is available on RHEL 9.

The repo named: codeready-builder-for-rhel-9-x86_64-rpms 
Should be enabled. It is under default redhat.repo file.

So basically, on RHEL9 the role should check if the codeready-builder-for-rhel-9-x86_64-rpms is ENABLED in order to pass the dependency packages installation step.

(subscription-manager repos --enable codeready-builder-for-rhel-9-$(arch)-rpms)

**Comment by wookietreiber** [comment]
@nortex without having to fully enable the repo (which is something I wouldn't want to impose with the role), does `yum install --enablerepo=codeready-builder-... lua-devel` work? If so, we could split the vars between RHEL 9 and Rocky 9, so rocky gets crb enabled, while RHEL gets the codeready-builder enabled. I'd prefer that solution.

**Comment by wookietreiber** [comment]
We now have a RHEL 9 server. My guess is that it is `crb` for Rocky Linux 9, which we already automatically enable in the role, while on RHEL 9 it's called `codeready-builder-for-rhel-9-x86_64-rpms`. Maybe `crb` is a shorthand for `codeready-builder`.

**Comment by wookietreiber** [comment]
Confirmed: 

```console
$ cat /etc/system-release
Red Hat Enterprise Linux release 9.1 (Plow)

$ yum list --enablerepo codeready-builder-for-rhel-9-x86_64-rpms lua-devel
Available Packages
lua-devel.i686            5.4.4-2.el9_1   codeready-builder-for-rhel-9-x86_64-rpms
lua-devel.x86_64          5.4.4-2.el9_1   codeready-builder-for-rhel-9-x86_64-rpms
```

```console
$ cat /etc/system-release
Rocky Linux release 9.1 (Blue Onyx)

$ yum list lua-devel
Last metadata expiration check: 1:41:31 ago on Mon 16 Oct 2023 12:49:19 PM CEST.
Installed Packages
lua-devel.x86_64   5.4.4-3.el9   @crb
```

I'll write a fix for this and will release a hotfix.

**Comment by wookietreiber** [comment]
released in v3.0.1, unfortunately, due to ansible galaxy limitations, currently only available via git source:

```yml
roles:
  - name: idiv_biodiversity.lmod
    src: https://github.com/idiv-biodiversity/ansible-role-lmod
    version: v3.0.1
```
